ext/search_kit/ang/crmSearchAdmin/compose.html

```diff
@@ -74,11 +74,11 @@
     </ul>
   </div>
 </div>
-<details id="crm-search-build-functions">
+<details id="crm-search-build-functions" class="crm-search-fieldTransforms">
   <summary ng-click="controls.showFunctions = !controls.showFunctions">
     {{:: ts('Field Transformations') }}
   </summary>
-  <div ng-if="!!controls.showFunctions">
+  <div ng-if="!!controls.showFunctions" class="crm-search-functions">
```

Comment:
And this is just an inner wrapper for the above so whatever name the outer div has the inner one should be similar.